src/shorthands/borderWidth.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import directionalProperty from '../helpers/directionalProperty'
+
+/**
+ * The border-width shorthand accepts up to four values, including null to skip a value, and uses the directional-property mixin to map them to their respective directions.
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...borderWidth('12px', '24px', '36px', '48px')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   ${borderWidth('12px', '24px', '36px', '48px')}
+ * `
+ *
+ * // CSS as JS Output
+ *
+ * div {
+ *   'border-top-width': '12px',
+ *   'border-right-width': '24px',
+ *   'border-bottom-width': '36px',
+ *   'border-left-width': '48px'
+ * }
+ */
+
+function borderWidth(...values: Array<string|null>) {
```

Comment:
If you set `null` as a type flow will think that's an `any`, which we need to avoid. Instead of saying that, do `param: ?string`. That'll tell flow that the param is null-able!